lib/ldclient-rb/stream.rb

```diff
@@ -157,12 +156,10 @@ def set_connected
 
     def should_fallback_update
       disc = @disconnected.get
-      disc != nil and disc < (Time.now - 120)
+      !disc.nil? && disc < (Time.now - 120)
```

Comment:
Do not use `Time.now` without zone. Use one of `Time.zone.now`, `Time.now.current`, `Time.now.in_time_zone`, `Time.now.utc`, `Time.now.getlocal`, `Time.now.iso8601`, `Time.now.jisx0301`, `Time.now.rfc3339`, `Time.now.to_i`, `Time.now.to_f` instead.